.github/workflows/deploy.yaml

```
@@ -208,9 +208,9 @@ jobs:
         run: curl https://${{ steps.runterraform.outputs.gaiax_did_host }}/.well-known/did.json | jq '.id'
 
       - name: 'Verify Dataspace Authority did endpoint is available'
```

nit: (feel free to ignore)
```suggestion
      - name: 'Verify Dataspace DiD endpoint is available'
```